models/action_recognition/model_templates/gesture-recognition/tools/data/crop_images.py

```diff
@@ -25,7 +27,8 @@
 import mmcv
 from tqdm import tqdm
 
-from openvino.inference_engine import IECore
+# TODO(eizutov): to fix importing non-existing module
```

Comment:
@evgeny-izutov

Reply:
`from openvino.inference_engine import IECore  # pylint: disable=no-name-in-module`
The code is Ok.

Reply:
ok